src/main/java/org/broadinstitute/hellbender/engine/FeatureWalker.java

```diff
@@ -96,6 +96,10 @@ public void traverse() {
                 });
     }
 
+    protected <T extends Feature> SimpleInterval makeFeatureInterval(final T feature) {
+        return new SimpleInterval(feature);
+    }
```

Comment:
What's the point of having this trivial method in the walker base class?

Reply:
Can you add javadoc explaining when someone might want to override this method?

Reply:
Done, but I am still open to reverting it.